openedx/core/djangoapps/edx_global_analytics/README.md

```diff
@@ -0,0 +1,88 @@
+# edX global analytics
+
+`EdX global analytics` is part of edx-platform as Django application. It has arisen out of [Open edX](https://open.edx.org)
+requirement to collect edx-platform installations statistics data from all over the world.
+
+So one at 24 hours this application fetches and sends edX installation statistics to [Acceptor](https://github.com/raccoongang/acceptor) developed by @raccoongang team.
+Acceptor is required to be able to collect, visualize and process this data. In general, it provides possibilities
+for analysing trends of platform usage and users engagement in e-learning process per country and globally around the world.
+It is usually carried out via graphs, world map and activity metrics.
+
+## The way sent data looks like on Acceptor
+
+![acceptor_world_map](https://user-images.githubusercontent.com/22666467/27970248-d015fd9c-6356-11e7-906c-8c052cd65b08.png)
+
+## Types of statistics sent, and dependence on statistics level
+
+Firstly, `statistic level` is a string constant in settings (`lms.env.json`) which regulates size of bunch platform sends.
+
+First `Paranoid` level with paranoid constant allows platform to transfer:
+1. Access token for Acceptor server.
+2. Active students amount per last calendar day, week and month*.
+3. Courses amount.
+
+Second `Enthusiast` level and the last one with `enthusiast` constant extends `Paranoid` level with:
+1. Platform URL and platform name.
+2. Platform's latitude and longitude.
+3. Active students per country accordance for last calendar day.
+
+\*active student is a student whose last login datetime value is included in calendar duration.
+
+## Settings
+
+Platform's administrator is able to configure `EdX global analytics` settings in `lms.env.json`.
+This settings already exist after default platform installation, admin just needs to change it.
+
+Settings context:
+```
+...,
+"OPENEDX_LEARNERS_GLOBAL_ANALYTICS": {
+    "CELERY_TIMEZONE": "Europe/Kiev",
+    "OLGA_ACCEPTOR_PERIODIC_TASK_URL": "",
+    "OLGA_ACCEPTOR_PERIODIC_TASK_URL_LOCAL_DEV": "http://192.168.1.10:7000",
+    "PLATFORM_CITY_NAME": "Kiev",
+    "STATISTICS_LEVEL": "enthusiast"
+}, ...
+```
+
+`CELERY_TIMEZONE`: Time zone in edx-platform is enabled by default, platform's administrator can set time zone in `lms.env.json` for `TIME_ZONE` key. `EdX global analytics` supports own time zone setting — time zone platform will fetch and send statistics to Acceptor.
```

Comment:
Why we need our own timezone? Maybe we can remove this at all, please explain, i do not understand it.